### ISSUE TITLE
Implementing Navigation Buttons and Logout Functionality in Vlearnup

### ISSUE THREAD

**Issue by VishalAadiith** [VishnuAmit/VLearnUp-ELearningHub#9]
### **Project Overview**
This project focuses on enhancing the navigation experience within the Vlearnup e-learning platform. You are tasked with creating buttons for key sections of the platform and implementing a logout feature to direct users appropriately.

### **Your Role**
1. Clone the repository and run the project.
2. Install the necessary packages using `npm install`.
3. Start the application using `npm start`.
4. Once started, sign up and go to the dashboard.

### **Navigation Buttons**
You will implement the following buttons in the navigation bar:

- **Home Button**:
  - Create a button that, when clicked, directs the user to the main page.

- **Courses Button**:
  - Create a button that, when clicked, scrolls the user down to the "Explore Courses" section of the page.

- **Pricing Button**:
  - Create a button that, when clicked, scrolls the user down to the "Pricing" section of the page.

- **Explore Button**:
  - Create a button that, when clicked, scrolls the user down to the "Our Features" section of the page.

- **About Us Button**:
  - Create a button that, when clicked, directs the user to the "About Us" page.

### **What Needs to be Done**
- Ensure all buttons are styled appropriately for a consistent look and feel across the platform.
- Test each button to verify that they direct users to the correct sections or pages.

If you have any questions, feel free to reach out!

**Comment by satyam1024** [comment]
Hi @VishnuAmit 
I would like to work on this issue.

**Comment by VishnuAmit** [comment]
Hey @satyam1024 . Good luck, You have been assigned.

**Comment by satyam1024** [comment]
Hi @VishnuAmit,
I'm almost finished, but I'm not sure which section corresponds to the pricing. Could you please help me identify it?
Thank you!

**Comment by VishnuAmit** [comment]
Hi @satyam1024 
Remove home button.
Remove dropdown list (downward arrow)  and that navigates to explore section if clicked on it.
Rewrite pricing as articles and navigates to latest blogs and insights section.
Explore - About our features.
About us - What is vlearnup section.

When  hovering over the button, the color should change the color: #F48C06;( Use this)

I hope this clears your doubt.

**Comment by satyam1024** [comment]
Is this what you have in mind
![Screenshot (3004)](https://github.com/user-attachments/assets/120154ab-d3a6-4f31-b2f6-af2534b1282d)

**Comment by VishnuAmit** [comment]
No, except login and sign up buttons, other should not have the boxes. 
Just keep it as the same, but when we hover it the color should change and go to the respective section.

**Comment by satyam1024** [comment]
something like this?
![Screenshot (3006)](https://github.com/user-attachments/assets/6b9bec7c-997a-4b48-9949-ab649a58e760)

**Comment by VishnuAmit** [comment]
Yea. Exactly.

**Comment by satyam1024** [comment]
Okay , i will be creating a pull request then
